html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>
+        Email
+      </div>
+      <div>
+        <label for="email">
```

Comment:
This is close. We want to have the text `Email` within the label. Something like this: 

```
<label for="email-address">Email</label>
<div>
  <input name="email" type="email" id="email-address">
</div>
```